extensions/amp-base-carousel/0.1/test-e2e/test-rtl.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  getNextArrow,
+  getPrevArrow,
+  getSlide,
+} from './helpers';
+
+const pageWidth = 800;
+const pageHeight = 600;
+
+describes.endtoend('AMP carousel', {
```

Comment:
nit: call this "AMP carousel rtl" or something else to differentiate from test-carousel.js